services/frontend-service/src/ui/utils/store.tsx

```diff
@@ -1557,3 +1557,21 @@ class GitSyncStatusGetter {
         return mostInteresting;
     }
 }
+
+export const GetTargetFutureDate = (current: Date | undefined, increment: string): Date | undefined => {
```

Comment:
It doesn't seem correct to put this function in the `store.tsx`. It doesn't have anything do with `storing` things:)
I would say for now we can put it in `LockDisplay.tsx`.